src/Features/Analytics.php

```diff
@@ -129,72 +132,83 @@ public function register_pages() {
 				'title'  => __( 'Overview', 'woocommerce-admin' ),
 				'parent' => 'woocommerce-analytics',
 				'path'   => '/analytics/overview',
+				'order'  => 10,
 			),
 			array(
 				'id'     => 'woocommerce-analytics-revenue',
 				'title'  => __( 'Revenue', 'woocommerce-admin' ),
 				'parent' => 'woocommerce-analytics',
 				'path'   => '/analytics/revenue',
+				'order'  => 20,
 			),
 			array(
 				'id'     => 'woocommerce-analytics-orders',
 				'title'  => __( 'Orders', 'woocommerce-admin' ),
 				'parent' => 'woocommerce-analytics',
 				'path'   => '/analytics/orders',
+				'order'  => 30,
 			),
 			array(
 				'id'     => 'woocommerce-analytics-products',
 				'title'  => __( 'Products', 'woocommerce-admin' ),
 				'parent' => 'woocommerce-analytics',
 				'path'   => '/analytics/products',
+				'order'  => 40,
```

Comment:
This demo has `Products` under `Overview`, https://cdpn.io/jameskoster/debug/wvKrmwv#

Reply:
Updated- this also updates in WCA, but I assume we'd want to keep consistent.